packages/app/src/app/pages/Sandbox/Editor/Navigation/Navigation.tsx

```diff
@@ -139,7 +139,7 @@ export const Navigation: FunctionComponent<Props> = ({
     >
       <SingletonTooltip placement="right">
         {singleton => (
-          <>
```

Comment:
I think we can't change this one, as the Container is a flexbox. But it shouldn't be a problem to make this `<></>` as `WorkspaceItems` will always be filled.